build.gradle

```diff
@@ -1029,6 +1029,7 @@ project(':metadata') {
     compile project(':clients')
     compile libs.jacksonDatabind
     compile libs.jacksonJDK8Datatypes
+    compile libs.metrics
```

Comment:
Let's file a JIRA to discuss whether the metadata module should use Kafka metrics or Yammer metrics.

Reply:
> file a JIRA to discuss whether the metadata module should use Kafka metrics or Yammer metrics.

https://issues.apache.org/jira/browse/KAFKA-12348

cc: @cmccabe

Reply:
thanks. I think this will be a post-2.8 thing